Jenkinsfile.nightly

```diff
@@ -49,14 +49,21 @@ pipeline {
 
         stage('Kubernetes tests') {
             steps {
-                parallel(
-                    "K8s-1.7":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.7 ginkgo --focus=" K8s*" -v -noColor'
-                    },
-                   "K8s-1.8":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.8 ginkgo --focus=" K8s*" -v -noColor'
-                    }
-                )
+                script {
+                    parallel(
+                        "K8s-1.7":{
+                            sh 'cd ${TESTDIR}; K8S_VERSION=1.7 ginkgo --focus=" K8s*" -v -noColor'
```

Comment:
Just wondering, how much load can the Jenkins agents handle w.r.t. concurrently running VMs?

Reply:
More than four servers things are getting slower.

Reply:
Great, thanks!